lib/src/components/NxCheckbox/__tests__/NxCheckbox.test.tsx

```diff
@@ -144,4 +144,11 @@ describe('NxCheckbox', function() {
     expect(component).toHaveProp('id', 'foo');
     expect(component).toHaveProp('htmlFor', 'baz');
   });
+
+  it('pass attributes into checkbox input element', function() {
```

Comment:
"it pass attributes into checkbox input element" is not grammatically correct

Reply:
> "it pass attributes into checkbox input element" is not grammatically correct

Fixed: 4c796beb23b05878976a76dfdcf72610774430aa